website/docs/d/oracle_db_system_shapes.html.markdown

```diff
@@ -0,0 +1,88 @@
+---
+subcategory: "Oracle"
+layout: "azurerm"
+page_title: "Azure Resource Manager: Data Source: azurerm_oracle_db_system_shapes"
+description: |-
+  Provides the list of Db System Shapes.
+---
+
+# Data Source: azurerm_oracle_db_system_shapes
+
+This data source provides the list of Db System Shapes in Oracle Cloud Infrastructure Database service.
+
+Gets a list of the shapes that can be used to launch a new DB system. The shape determines resources to allocate to the DB system - CPU cores and memory for VM shapes; CPU cores, memory and storage for non-VM (or bare metal) shapes.
+## Example Usage
+
+```hcl
+data "azurerm_oracle_db_system_shapes" "example" {
+  location = "West Europe"
+}
+
+output "id" {
+  value = data.azurerm_oracle_db_system_shapes.example.id
+}
+```
+
+## Arguments Reference
+
+The following arguments are supported:
+
+* `location` - (Required) The Azure Region to query for the system shapes in.
+
+## Attributes Reference
+
+In addition to the Arguments listed above - the following Attributes are exported: 
+
+* `id` - The ID of the Oracle.
+
+* `db_system_shapes` - A `db_system_shapes` block as defined below.
+
+---
+
+A `db_system_shapes` block exports the following:
+
+* `available_core_count` - The maximum number of CPU cores that can be enabled on the DB system for this shape.
+
+* `available_core_count_per_node` - The maximum number of CPU cores per database node that can be enabled for this shape. Only applicable to the flex Exadata shape, ExaCC Elastic shapes and VM Flex shapes.
```

Comment:
Ok It seems `ExaCC` is something Oracle specific, can you please correct the casing in the other doc so that it's clear and consistent?